hs_discover/src/components/Resources.vue

```diff
@@ -1,627 +0,0 @@
-<template>
```

Comment:
Older version of Discover used Resources as a Component, now removing vestigial parent and flattening into one vue component